open-api/rest-catalog-open-api.py

```diff
@@ -132,11 +132,15 @@ class ExpressionType(BaseModel):
 
 
 class TrueExpression(BaseModel):
-    type: ExpressionType
+    type: ExpressionType = Field(
+        default_factory=lambda: ExpressionType.parse_obj('true'), const=True
+    )
```

Comment:
This still looks a bit odd, but I think we can refactor this by inlining the `ExpressionType`, but I didn't want to do that in this PR. I think this is better than before.

Reply:
It makes sense to do it in another PR (not related to the update).